oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/query/input/Duration.java

```diff
@@ -27,45 +27,50 @@
 
 @Getter
 @Setter
+/**
+ * To optimize the query, the range of start and end times will be trimmed to [TTL_deadLine < time <= CurrentTime].
+ */
```

Comment:
```suggestion
/**
 * @since 9.3.0 To optimize the query, the range of start and end times will be trimmed to [TTL_deadLine < time <= CurrentTime].
 */
@Getter
@Setter
```

This seems a better comment style?